deployment/helm/node-feature-discovery/templates/worker.yaml

```
@@ -122,6 +125,9 @@ spec:
         - name: host-lib
           hostPath:
             path: "/lib"
+        - name: host-proc-swaps
+          hostPath:
+            path: "/proc/swaps"
```

We could (or should) also start using types here

```suggestion
            path: "/proc/swaps"
            type: File
```
WDYT?

---

I don't really have a preference here. My main thing is that this could be done in one go for all the other types.

---

That's fine. Probably cleaner to do it in one go for all